fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -50,8 +56,6 @@ public enum StorageVolumeType {
 
     private String comment;
 
-    private int refCount;
-
     private boolean enabled;
 
     private boolean isDefault;
```

Comment:
There already has `defaultSV` in the storage volume mgr, do we still need to set `isDefault` in the storage volume?